language/intro_prompt_design.ipynb

```diff
@@ -1,739 +1,1124 @@
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "negative\n"
+          ]
+        }
+      ],
+      "source": [
+        "prompt = \"\"\"Decide whether a Tweet's sentiment is positive, neutral, or negative.\n",
+        "\n",
+        "Tweet: I loved the new YouTube video you made!\n",
+        "Sentiment: positive\n",
+        "\n",
+        "Tweet: That was awful. Super boring 😠\n",
+        "Sentiment:\n",
+        "\"\"\"\n",
+        "\n",
+        "print(generation_model.predict(prompt=prompt, max_output_tokens=256).text)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ef58c35005c0"
+      },
+      "source": [
+        "#### Few-shot prompt"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b630e8947b60"
+      },
+      "source": [
+        "Below is an example of few-shot prompting, where you provide one example to the LLM within the prompt to give some guidance on what type of response you want."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 22,
+      "metadata": {
+        "id": "fb3ba21bbd11",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "ef090157-0c14-405b-f8ea-41fc3aab7ee7"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "positive\n"
+          ]
+        }
+      ],
+      "source": [
+        "prompt = \"\"\"Decide whether a Tweet's sentiment is positive, neutral, or negative.\n",
+        "\n",
+        "Tweet: I loved the new YouTube video you made!\n",
+        "Sentiment: positive\n",
+        "\n",
+        "Tweet: That was awful. Super boring 😠\n",
+        "Sentiment: negative\n",
+        "\n",
+        "Tweet: Something surprised me about this video - it was actually original. It was not the same old recycled stuff that I always see. Watch it - you will not regret it.\n",
+        "Sentiment:\n",
+        "\"\"\"\n",
+        "\n",
+        "print(generation_model.predict(prompt=prompt, max_output_tokens=256).text)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "a4023be726eb"
+      },
+      "source": [
+        "#### Choosing between zero-shot, one-shot, few-shot prompting methods"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6d7870ff75cc"
+      },
+      "source": [
+        "Which prompt technique to use will solely depends on your goal. The zero-shot prompts are more open-ended and can give you creative answers, while one-shot and few-shot prompts teach the model how to behave so you can get more predictable answers that are consistent with the examples provided."
+      ]
+    },
+    {
+      "cell_type": "code",
```

Comment:
execution count 22, is empty and should be removed

Reply:
execution count 22 has been removed